src/contexts/Pools/defaults.ts

```diff
@@ -13,3 +13,11 @@ export const stats = {
   minCreateBond: new BN(0),
   minJoinBond: new BN(0),
 };
+
+export const poolBondOptions = {
```

Comment:
Since pool bonds are tied to staking bonds, this should be the same default as bondOptions. Should we introduce a global `src/defaults.ts` file for things like this?

Reply:
Might be a good idea, but I leave it for now in case we see some other values need to be added here. If they look identical after the all nomination pools flows are added, we can refactor and move them to a more generic module.